com.parallax.PropellerIDE.svg

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I'm surprised it's not part of upstream code and I guess it should also be upstreamed.